webservice/dispatcher.php

```diff
@@ -28,7 +28,13 @@
 
 ob_start();
 
-require_once dirname(__FILE__).'/../config/config.inc.php';
+require_once dirname(__FILE__, 1) . '/config/config.inc.php';
```

Comment:
On windows and php 7.3, this require doesn't work : with dirname(__FILE__, 2) it's ok, so I suggest to go back to the original.

Reply:
dirname 2 is the way to go I haven't check the suggestion first, it should work now